src/transform/src/normalize_lets.rs

```diff
@@ -596,7 +590,7 @@ mod let_motion {
     }
 
     /// Harvest any safe-to-lower non-recursive suffix of binding from a
-    /// `LetRec` expression.
+    /// `LetRec` expression, potentially wrapped in a stack of `Let`s.
```

Comment:
This comment change is a bug now that it is doing things another way. Will fix before merging, but don't want to kick off a full CI run just for this.